### PR TITLE
PR for release 1.0.0

### PR DESCRIPTION
**Pull request by ypriverol**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/pgdb/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/pgdb _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,17 +53,13 @@ By default, the pipeline currently performs the following:
...
 The nf-core/pgdb pipeline comes with documentation about the pipeline: [usage](https://nf-co.re/pgdb/usage) and [output](https://nf-co.re/pgdb/output).
 
-<!-- TODO nf-core: Add a brief overview of what the pipeline does and how it works -->
-
 ## Credits
 
 nf-core/pgdb was originally written by Husen M. Umer & Yasset Perez-Riverol.
 
 We thank the following people for their extensive assistance in the development
 of this pipeline:
```
Comment (Leon-Bichmann) [inline README.md:65]: Maybe add contributor names here

File: README.md [hunk 2]
```diff
@@ -86,7 +76,3 @@ You can cite the `nf-core` publication as follows:
 >
 > _Nat Biotechnol._ 2020 Feb 13. doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x).
 > ReadCube: [Full Access Link](https://rdcu.be/b1GjZ)
```
Comment (Leon-Bichmann) [inline README.md:81]: If not too much work you can add references to the components of the pipelines, to make it easier for users to find the corresponding tools it is using.

File: bin/scrape_software_versions.py [hunk 3]
```diff
@@ -3,12 +3,9 @@
 from collections import OrderedDict
 import re
 
-# TODO nf-core: Add additional regexes for new tools in process get_software_versions
 regexes = {
     "nf-core/pgdb": ["v_pipeline.txt", r"(\S+)"],
     "Nextflow": ["v_nextflow.txt", r"(\S+)"],
```
Comment (Leon-Bichmann) [inline bin/scrape_software_versions.py:8]: Maybe pypgatk could be added here

File: README.md (hunk 2 above)
Comment (Leon-Bichmann) [reply README.md:81]: I guess pypgatk is the most important one here

File: docs/usage.md [hunk 4]
```diff
@@ -6,8 +6,6 @@
 
 ## Introduction
 
-<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
-
 ## Running the pipeline
 
 The typical command for running the pipeline is as follows:
```
Comment (Leon-Bichmann) [inline docs/usage.md]: Maybe adapt to same as in the readme - there the typical command was:

    ```bash
    nextflow run nf-core/pgdb -profile <docker/singularity/podman/conda/institute> --ensembl_name homo_sapines --ensembl false
    ```

File: conf/test_full.config [hunk 5]
```diff
@@ -12,8 +12,6 @@ params {
   config_profile_description = 'Full test COSMIC generation'
 
   // Input data for full size test
```
Comment (Leon-Bichmann) [inline conf/test_full.config:14]: Maybe the other functions of the pipeline would need a test case as well for example the vcf input or the various options that can be selected.

File: README.md [hunk 6]
```diff
@@ -56,17 +53,13 @@ By default, the pipeline currently performs the following:
```
Comment (Leon-Bichmann) [inline README.md:59]: Ok so by reading the README im not exactly sure which variants it includes for example from COSMIC. Does it include them all?

Comment (Leon-Bichmann) [reply README.md:59]: Must be quite a large DB in the end

File: README.md [hunk 7]
```diff
@@ -12,7 +12,6 @@ The ProteoGenomics database generation workflow (**pgdb**) use the [pypgatk](htt
 
 ## Introduction
 
-<!-- TODO nf-core: Write a 1-2 sentence summary of what data the pipeline is for and what it does -->
 **nf-core/pgdb** is a bioinformatics best-practise analysis pipeline for
```
Comment (Leon-Bichmann) [inline README.md]: I think this sentence must be extended a bit and explain a bit clearer what the default pipeline call does.

**Review by Leon-Bichmann — CHANGES_REQUESTED** [review]
Ok overall it looks good and I think this is quite a usefull pipeline for proteogenomics! If this becomes a DSL2 module at some point it can also be nicely combined with the other proteomics pipelines.

I made a few comments mainly on documentation and readme. Finally the tests could be expanded quite a bit but this can be done in a future PR as well. Multiple different test cases can then also be run in parallel by the github actions.

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,12 +3,23 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## v1.0.0 - Lovely Logan [18.10.2020]
```
Comment (jpfeuffer) [inline CHANGELOG.md]: please change ;)

File: conf/protein_decoy.yaml [hunk 9]
```diff
@@ -9,7 +9,6 @@ proteindb_decoy:
   keep_target_hits: true
```
Comment (jpfeuffer) [inline conf/protein_decoy.yaml:9]: If those params were part of the pipeline you could validate and display them in a UI. But could be done in a future version.

File: README.md (hunk 2 above)
Comment (ypriverol) [reply README.md:81]: @Leon-Bichmann pypgatk hasn't been published.

**Review by jpfeuffer — CHANGES_REQUESTED** [review]
See my requested changes. And all of Leon's are important too.
The actual pipeline I will check on the PR to master.

**Comment by jpfeuffer** [comment]
P.S: Why didn't you change to the new help generation introduced with the new template? Or am I missing it?
You can remove your help message now and add a small "createHelp" command that reads from the json schema

File: conf/test_full.config (hunk 5 above)
Comment (ypriverol) [reply conf/test_full.config:14]: We have tried to add more tests but the size in Github for most of those databases is huge.

File: README.md (hunk 7 above)
Comment (ypriverol) [reply README.md]: Thanks a lot, the README is now better.

**Review by Leon-Bichmann — CHANGES_REQUESTED** [review]
The changes look good to me, just noticed one more small thing:
 - In the nf-core schema the parameters are still mostly ungrouped
 - and the parameter descriptions are missing, you could just copy them from the main.nf

You could create parameter groups with `nf-core schema build pgdb/` to make the usage docs that will be displayed on nf-co.re/pgdb much easier to be overviewed.

**Comment by Leon-Bichmann** [comment]
I think the linting is failing because you accidently did a change to:
.github/PULL_REQUEST_TEMPLATE.md and lib/NfcoreSchema.groovy

Maybe you can replace them with the originals from the TEMPLATE branch again.

**Review by Leon-Bichmann — APPROVED** [review]
ok great, LGTM!

**Review by jpfeuffer — APPROVED** [review]
Yes looks much better already.

**Pull request merged**